### ISSUE TITLE
Docs: Describe which errors show up in the Dashboard and which not

### ISSUE THREAD

**Issue by SmallhillCZ** [traefik/traefik#10148]
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.traefik.io) and didn't find any.

### What did you expect to see?

It is unclear from the docs and/or the website screenshots what errors get shown in the Dashboard UI.

For example:

1) If I mistype the param name in Docker router label, the router **will not show up at all** without any hint what went wrong in Dashboard (the error is only in the logs)
```yaml
labels:
  traefik.http.routers.my-router.rul: Host(`example.com`) # see missing "e" in rule
```

2) If I mistype the param value in Docker label, the router **will show up with an error** in Dashboard
```yaml
labels:
  traefik.http.routers.my-router.rule: Host(`example.com` # see missing trailing ")"
```

3) If I mistype certResolver value in Docker tls label, the router **will show up without any error** in Dashboard (the error is only in the logs)
```yaml
labels:
  traefik.http.routers.my-router.tls.certresolver: non-existing-resolver
```

**Comment by ldez** [comment]
Hello,

typos in the name of the options are reported to logs. (because this a structural configuration error)
errors related to the value of the options are reported in the logs and the dashboard. (because it's related to the interpretation of the value.)

The 2 checks are not at the same moment: one is before we know what a router, a service, etc., one this during the "build" of a router, a service, etc.

**Comment by nmengin** [comment]
Hello @SmallhillCZ,

Thanks for reaching out.

Have you checked the following sections in the documentation:
- https://doc.traefik.io/traefik/getting-started/faq/#what-does-the-field-not-found-error-mean
- https://doc.traefik.io/traefik/getting-started/faq/#why-are-some-resources-routers-middlewares-services-not-createdapplied

If you don't find enough information, could you indicate the improvements we could bring?

**Comment by ldez** [comment]
@nmengin your answer is offtopic: the issue is not about the errors meaning but the dashboard.

**Comment by rtribotte** [comment]
@ldez @tomMoulard I don't think this is off-topic, and I agree with @nmengin's message.
Basically, the idea behind it is to identify what is missing in our FAQ.
In this documentation section, we are addressing troubleshooting hints, and hence enhancing it in regards to which errors could be seen in the dashboard or not, and why, which seems to be a good idea to me.

**Comment by ldez** [comment]
The content of my [first comment](https://github.com/traefik/traefik/issues/10148#issuecomment-1743116948) is the answer of the questions:

> It is unclear from the docs and/or the website screenshots what errors get shown in the Dashboard UI.

> Describe which errors show up in the Dashboard and which not

**Comment by rtribotte** [comment]
> The content of my https://github.com/traefik/traefik/issues/10148#issuecomment-1743116948 is the answer of the questions:

Indeed, and this is why this issue looks like a documentation enhancement request, and why asking (from @SmallhillCZ point of view) for what is missing/not clear in the current FAQ could be useful as well.

> To use FAQ, with an error message, you need to know where are the messages.

I'm not sure to understand, I'm not suggesting we should document every error in one place.
It feels like, currently, the FAQ addresses general help, and we could improve it by indicating where you should be looking for errors while troubleshooting.

**Comment by SmallhillCZ** [comment]
Yes – despite IMO it would be nice if all of these errors showed in the dashboard – the point of this issue was to have it documented in the Traefik docs, that different types errors show up in different places and user should therefore check those and not expect them to show up in the Dashboard.

For example I didn't know what @ldez wrote in his comment and didn't find it anywhere in the docs, so I spent the entire day figuring out why they don't show in the dashboard :)
